app/views/sign_up/completions/_requested_attributes.html.slim

```diff
@@ -0,0 +1,5 @@
+ul.mb3.list-reset.border-bottom.success-bullets.requested-attributes
+  - view_model.requested_attributes_sorted.each do |attribute|
+    - if decorated_session.requested_attributes.include? attribute
```

Comment:
If we passed the decorated session (or even just the regular session) into the view model, I bet we could move all the logic there? For one method like `sorted_requested_attributes`